src/components/displayTable/DataDisplay.tsx

```diff
@@ -61,7 +57,9 @@ const DataDisplay: React.FC<DataDisplayProps> = ({ getData }) => {
           pageCount={Math.ceil(totalItems / itemsPerPage)}
           marginPagesDisplayed={2}
           pageRangeDisplayed={5}
```

Comment:
what is this hardcoded number 5

Reply:
It displays the number of pages . for example if there are total 10 pages and you are 3rd page the UI will look something like 
`previous 1 2 3 4 5 ... next`